chroma_core/lib/job.py

```diff
@@ -84,6 +84,7 @@ def satisfied(self):
             depended_object = self.get_stateful_object()
         except:
             self.stateful_object.__class__._base_manager.get(pk=self.stateful_object.pk)
+
```

Comment:
Can you extract the changes in this file to it's own patch? They can land separately from this work.

Reply:
https://github.com/whamcloud/integrated-manager-for-lustre/pull/1025